proto/cosmos/distribution/v1beta1/genesis.proto

```diff
@@ -112,7 +112,6 @@ message GenesisState {
   option (gogoproto.equal)           = false;
   option (gogoproto.goproto_getters) = false;
 
-  // params defines all the paramaters of the module.
```

Comment:
why remove the comment? I think it's still useful

Reply:
huh? It's just a typo fix.

Reply:
updated in 55455728b78dbf9460ebdc1ae03dc74b758f92a7